.github/workflows/build-and-test.yml

```diff
@@ -68,7 +68,9 @@ jobs:
         run: |
           ./docker/dev.sh -d
           sleep 30
-          ./smoke-test/smoke.sh
+          ./smoke-test/smoke.sh || true
+          docker ps -a
+          docker logs datahub-frontend-react
```

Comment:
let's revert this now that smoke test seems to be working?

Reply:
yes